src/System.Management.Automation/FormatAndOutput/common/DisplayDatabase/typeDataXmlLoader_Table.cs

```diff
@@ -353,7 +353,7 @@ private TableRowDefinition LoadRowEntryDefinition(XmlNode rowEntryNode, int inde
             using (this.StackFrame(rowEntryNode, index))
             {
                 bool appliesToNodeFound = false;    // cardinality 0..1
-                bool columnEntriesNodeFound = false;         // cardinality 1
+                const bool columnEntriesNodeFound = false;         // cardinality 1
```

Comment:
The change causes CI fail because code unreachable after line 379. Possible bug.